core/service/storage-service.js

```diff
@@ -59,6 +59,38 @@ var StorageService = exports.StorageService = Montage.specialize({
         }
     },
 
+    getVolumeSnapshotData: {
```

Comment:
Some fixes are needed on naming (it can also be applied to following methods in this class):

- replace `get` with `list` (`get` is to be used only when the method returns a single value)
- `Data` is useless and confusing (the method has nothing to do with the data)
- use plural for methods that return an array